src/Middleware/Diagnostics/src/DeveloperExceptionPage/Views/ErrorPageModel.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable
```

Comment:
💡 A comment might be helpful

Reply:
We have a number of these internal types that I haven't been annotating in this pass and that's well understood internally. We'll look at updating these piecemeal.